src/test/java/edu/kis/vh/nursery/RyhmersJUnitTest.java

```diff
@@ -4,68 +4,42 @@
 import org.junit.Test;
 
 public class RyhmersJUnitTest {
-
+    private final int EMPTY_STACK_VALUE = -1;

```

Comment:
Original constant should be used (it is better to use a constant from stacks interface).